### PR TITLE
Fix not-registered env

### PR DESCRIPTION
**Pull request by modanesh**
## Description
In the `tests/test_run.py`, the `Pendulum-v0` is changed to `Pendulum-v1`. Because of the required gym version, there are no `Pendulum-v0` registered, so it throws an error. That is also the same case for `tests/test_utils.py`, where the `BreakoutNoFrameskip-v5` is used while the only two valid versions are: `BreakoutNoFrameskip-v0` and `BreakoutNoFrameskip-v4`.

## Motivation and Context
Fixing test errors of PR #572 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] I've read the [CONTRIBUTION](https://github.com/DLR-RM/stable-baselines3/blob/master/CONTRIBUTING.md) guide (**required**)
- [x] I have updated the changelog accordingly (**required**).
- [ ] My change requires a change to the documentation.
- [x] I have updated the tests accordingly (*required for a bug fix or a new feature*).
- [ ] I have updated the documentation accordingly.
- [ ] I have reformatted the code using `make format` (**required**)
- [ ] I have checked the codestyle using `make check-codestyle` and `make lint` (**required**)
- [x] I have ensured `make pytest` and `make type` both pass. (**required**)
- [ ] I have checked that the documentation builds using `make doc` (**required**)

Note: You can run most of the checks using `make commit-checks`.

Note: we are using a maximum length of 127 characters per line

### CODE REVIEW

**Comment by Miffyli** [comment]
Hey. For completeness sake, could you fill in the PR template with link to the said PR?

**Comment by modanesh** [comment]
The PR description is now updated according to the given template.

**Comment by araffin** [comment]
Hello,
you should probably do that PR to @jkterry1 fork: https://github.com/jkterry1/stable-baselines3
so we keep only one PR.
I would also wait anyway before merging as doc/tutorial need to be updated too (see comment  https://github.com/DLR-RM/stable-baselines3/pull/572#issuecomment-993703535) and cap gym version as additional breaking changes have been made (https://github.com/openai/gym/issues/2531).

**Comment by jkterry1** [comment]
@araffin I asked modanesh to create an updated PR based on mine to take over getting this merged, I closed my PR. 

@modanesh can you please update all the docs like antonin requested, update the branch to master, make sure tests pass, and cap the version of gym at the currently release one? (we can do more PRs for future versions of Gym as they release)

@araffin is there anything else that needs to be done here?

**Comment by vwxyzjn** [comment]
Hey just a quick note: https://github.com/openai/gym/issues/2531 was an issue with the gym's master and I have not encountered any issues with the [`0.21.0` release](https://github.com/openai/gym/releases/tag/v0.21.0). You can still use things like `BreakoutNoFrameskip-v4`. One thing you do have to do though is to do `pip install AutoROM[accept-rom-license]` instead of the `atari-py`, but that's about it.

Perhaps it's worth merging this PR as is, and putting adopting ALE-v5 environments and docs changes to a separate PR?

File: setup.py [hunk 1]
```diff
@@ -116,7 +116,7 @@
             # For render
             "opencv-python",
             # For atari games,
-            "atari_py~=0.2.0",
+            "ale-py~=0.7",
```
Comment (vwxyzjn) [inline setup.py:119]: Could we do 

```
    "ale-py~=0.7.1",
    "autorom[accept-rom-license]~=0.4.2",
```

**Review by vwxyzjn — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (araffin) [reply setup.py:119]: I think we should include autorom yes, otherwise it will be confusing.

**Comment by jkterry1** [comment]
@araffin this can be closed in favor of #734, modanesh seems to have quit

**Comment by AdamGleave** [comment]
Closing in favor of #734

**Pull request closed without merging**